api/geth_backend.go

```diff
@@ -697,7 +697,7 @@ func (b *GethStatusBackend) getVerifiedWalletAccount(address, password string) (
 		b.log.Error("failed to get a selected account", "err", transactions.ErrInvalidTxSender)
 		return nil, transactions.ErrAccountDoesntExist
 	}
-
+    b.accountManager.ReadKey = true
```

Comment:
I guess whether to read a key or not should come from a caller so a caller of `getVerifiedWalletAccount`, right? Eventually, we want the mobile app to decide whether to return a key or now.

I would change the interface of `accountManager` to be explicit:
```go
func (b *GethStatusBackend) verifyWalletAccount(address, password string) error {
    // ...
    return b.accountManager.VerifyAccount(config.KeyStoreDir, address, password)
}

func (b *GethStatusBackend) verifyAndReturnWalletAccount(address, password string) (*account.SelectedExtKey, error) {
    // ...
    key, err := b.accountManager.VerifyAccountAndReturnKey(config.KeyStoreDir, address, password)
    // ...
}
```

Reply:
> from a caller so a caller of getVerifiedWalletAccount, right?

Ok sure, will make the change